Header header
uint8 landmark_number  
uint8[20] landmark_type     	# 1 is X, 2 is T, 3 is L, 4 is unknown, 5 is penaltypoint
float64[20] landmark_range                 
float64[20] landmark_bearing               
float32[20] landmark_confidence               # confidence 0..1
